integration/e2e/pages/viewer.po.ts

```diff
@@ -23,4 +27,103 @@ export class ViewerPage {
     utils.waitForElement(el);
     return el;
   }
+
+  /*
+  Getters & Setters
+   */
+  getAnimationTime(): promise.Promise<number> {
+    return browser.executeScript('return window.openSeadragonViewer.animationTime;');
+  }
+
+  getHomeZoom(): promise.Promise<number> {
+    return browser.executeScript('return window.openSeadragonViewer.viewport.getHomeZoom();');
+  }
+
+  setHomeZoom(): promise.Promise<any> {
+    return browser.executeScript('window.openSeadragonViewer.viewport.goHome(true);');
+  }
+
+  getZoomLevel(): promise.Promise<number> {
+    return browser.executeScript('return window.openSeadragonViewer.viewport.getZoom(true);');
+  }
+
+  setZoomLevel(level: number): promise.Promise<any> {
+    return browser.executeScript('window.openSeadragonViewer.viewport.zoomTo(' + level + ');');
+  }
+
+  getMinZoom(): promise.Promise<number> {
+    return browser.executeScript('return window.openSeadragonViewer.viewport.getMinZoom();');
+  }
+
+  getMaxZoom(): promise.Promise<number> {
+    return browser.executeScript('return window.openSeadragonViewer.viewport.getMaxZoom();');
+  }
+
+  /*
+  Actions
+   */
+  pinchOut(): promise.Promise<void> {
+    return browser.touchActions()
+      .tapAndHold(this.thumbStartPosition)
+      .tapAndHold(this.pointerPosition1)
+      .move(this.pointerPosition2)
+      .perform();
+  }
+
+  pinchIn(): promise.Promise<void> {
+    return browser.touchActions()
+      .tapAndHold(this.thumbStartPosition)
+      .tapAndHold(this.pointerPosition2)
+      .move(this.pointerPosition1)
+      .perform();
+  }
+
+  zoomIn(): promise.Promise<boolean> {
```

Comment:
This could be done like this:

async  zoomIn(): promise.Promise<boolean> {
  const currentZoomLevel = await this.getZoomLevel();
  const newZoomLevel = currentZoomLevel + 2;
  return browser.executeScript(`window.openSeadragonViewer.viewport.zoomTo(${newZoomLevel});`);
}